### ISSUE TITLE
Exception in Odoo Autocompletion Support

### ISSUE THREAD

**Issue by THEGAME999** [wt-io-it/odoo-pycharm-plugin#54]
Plugin Version: 0.5.16
IDEA Version: PyCharm 2022.2

<details><summary>java.lang.Throwable: Assertion failed</summary>

```
java.lang.Throwable: Assertion failed
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:220)
	...
	at com.intellij.psi.search.FilenameIndex.getFilesByName(FilenameIndex.java:79)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModuleDirectorySlow$3(OdooModuleServiceImpl.java:136)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:928)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectorySlow(OdooModuleServiceImpl.java:134)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectory(OdooModuleServiceImpl.java:121)
	at at.wtioit.intellij.plugins.odoo.modules.index.OdooDeserializedModuleImpl.getDirectory(OdooDeserializedModuleImpl.java:64)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:15)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:51)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	...
	at java.base/java.lang.Thread.run(Thread.java:833)
```
</details>

**Comment by bigbear3001** [comment]
@THEGAME999 can you update the plugin, this is the same as #33 and was fixed with version 0.5.18?

**Comment by THEGAME999** [comment]
Thanks
I will
